browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -70,6 +70,11 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
       base::make_span(kBraveWalletPageGenerated, kBraveWalletPageGeneratedSize),
       IDR_WALLET_PAGE_HTML);
   source->AddString("braveWalletLedgerBridgeUrl", kUntrustedLedgerURL);
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::ImgSrc,
+      base::JoinString({"img-src", "'self'", "chrome://resources",
+                        "chrome://erc-token-images", "chrome://image;"},
+                       " "));
```

Comment:
Let's not join these at runtime. Just a single string is enough.
10/10 random picks of OverrideContentSecurityPolicy in chrome use a constant string like here
https://source.chromium.org/chromium/chromium/src/+/main:ash/webui/projector_app/untrusted_projector_ui.cc;l=52-87?q=untrusted_projector_ui.cc&ss=chromium

Reply:
refactored to use not join the string at runtime 38533a7